lib/DistributionProvider.pm

```diff
@@ -35,14 +36,15 @@ If there is no matched version, then returns Tumbleweed as the default one.
 =cut
 
 sub provide {
+    return Distribution::Sle::AgamaDevel->new() if is_sle('>=15') && get_var('VERSION', '') =~ /agama/;
```

Comment:
agama devel is not sle 16, that will arrive back later.

Reply:
d5bcd77d7